src/lib/main.js

```diff
@@ -16,16 +18,16 @@ XPCOMUtils.defineLazyModuleGetter(this, 'console',
 XPCOMUtils.defineLazyModuleGetter(this, 'CustomizableUI',
   'resource:///modules/CustomizableUI.jsm');
 
-var EXPORTED_SYMBOLS = ['Main']; // eslint-disable-line no-unused-vars
+const EXPORTED_SYMBOLS = ['Main']; // eslint-disable-line no-unused-vars
 
-var onTabSelect = function() { console.log('onTabSelect'); };
-var onTabOpen = function() { console.log('onTabOpen'); };
-var onTabClose = function() { console.log('onTabClose'); };
+const onTabSelect = function() { console.log('onTabSelect'); };
```

Comment:
@pdehaan What do you think of this `const <function expression>` thing? I think it looks weird, and my brain sweats a little when I try to think about block vs lexical scoping _and_ function expression hoisting rules at the same time. I'm thinking it might be good (in a followup commit) to just use function declarations and generally outlaw function expressions. Thoughts?